SU-HSDO/suhumsci
# Shs 119 megamenu color pairing

## Summary
This adds color pairing to the new megamenu for child submenus.

## Steps to Test
1. On the Colorful site, make sure the 'Local Masthead Variant' is set to dark (http://hs-colorful.suhumsci.loc/admin/appearance/settings/humsci_colorful)
2. Ensure that the new mega menu is enabled (Site options: http://hs-colorful.suhumsci.loc/admin/config/site-options - you will need to flush the cache for this to take effect).
3. Visit any page and check out the menu at mobile size. With a dark masthead, the text color should be white for all menu item levels.
4. At desktop, level two items should appear as the color pairing color. You can see the color pairing color on the theme settings page, the same as the one linked above. Mine is set to 'Cardinal' as seen in the screenshot below.
5. Level three items should be black. 
6. Go back to the theme settings and change the 'Local Masthead Variant' to default. 
7. Visit your testing page again and view the menu at mobile. Level two items should be the color pairing color while level three retains black.
8. View the menu at desktop, it should follow the same pattern as step 6: level two color pairing color, level three black.
9. Repeat these steps for the traditional theme. You will likely have to enable the mega menu.

## Screenshots
![Screen Shot 2022-10-18 at 1 35 59 PM](https://user-images.githubusercontent.com/8405274/196506410-d68ad22a-2a59-446b-aac6-168d9ef1f656.png)

## Notes
I don't see a release branch for us yet, so I set it to Mike's release branch.
